salt/metalk8s/salt/master/installed.sls

```diff
@@ -57,7 +57,8 @@ Make sure salt master container is up:
 
 Wait for Salt API to answer:
   http.wait_for_successful_query:
-    - name: http://{{ salt_ip }}:4507/
+    - name: https://{{ salt_ip }}:4507/
```

Comment:
There is no convention about changing the port for https? Just asking

Reply:
Not that I know (even the 4507 is not a "standard" Salt API port)